packages/l10n/README.md

```diff
@@ -97,3 +97,265 @@ First, download the data (`core.zip`) for a specific version from the [downloads
 Then extract the data and copy the `core` folder to this package, and rename it to e.g. `cldr-v35`. Then point the `cldr` library to the folder location.
 
 Run the script, which uses the new data.
+
+## Utils
+
+### `applyTransformedLocalizedFields`
+
+> Transforms multiple `LocalizedField` -> `LocalizedString`, given `fieldNameTransformationMappings`
+
+#### Context
+
+Amongst [Common Types](https://docs.commercetools.com/api/types) in the commercetools platform API, we find [`LocalizedString`](https://docs.commercetools.com/api/types#localizedstring) type.
+
+The `LocalizedString` is a type reserved for values found in a [Resource](https://docs.commercetools.com/api/types#referencetype), for example `Product`:
+
+```js
+// Product
+{
+  // `name` is a `LocalizedString`
+  // as defined by https://docs.commercetools.com/api/projects/products#productdata
+  name: {
+    en: 'Milk';
+  }
+}
+```
+
+The documented `LocalizedString` in [https://docs.commercetools.com](https://docs.commercetools.com/) is a specification of the HTTP API.
+
+However, the commercetools platform `/graphql` API represents the `LocalizedString` in a different format as a list, for technical reasons.
```

Comment:
By technical reasons you mean constraints in how we can represent something like it in a GraphQL schema?

Reply:
the suggestion came from here
https://github.com/commercetools/merchant-center-application-kit/pull/1885#discussion_r527623681

in hindsight, I prefer the previous description. it's all technical and there is no need to be transparent about limitations